app-containers/podman/files/podman.initd

```diff
@@ -12,9 +12,26 @@ pidfile="${RUN_PATH}/${RC_SVCNAME}.pid"
 command="/usr/bin/podman"
 command_args="--log-level ${LOG_LEVEL} system service -t 0 ${SOCKET}"
 command_background="true"
-start_stop_daemon_args="--stdout ${LOG_PATH}/${RC_SVCNAME}.log --stderr ${LOG_PATH}/${RC_SVCNAME}.log --user ${RUN_AS_USER}"
+command_user="${RUN_AS_USER}"
+start_stop_daemon_args="--stdout ${LOG_PATH}/${RC_SVCNAME}.log --stderr ${LOG_PATH}/${RC_SVCNAME}.log"
 
-start() {
+extra_commands="start_containers"
+description_start_containers="Start containers with restart policy set to always"
+
+depend() {
+	need sysfs cgroups
+}
+
+start_containers() {
+	ebegin "Starting containers with restart policy set to always"
+	${command} start --all --filter restart-policy=always
```

Comment:
Can we split this into a separate service? I wonder if we should add a similar service unit for systemd.